met-web/src/components/landing/LandingComponent.tsx

```diff
@@ -189,8 +189,9 @@ const LandingComponent = () => {
                                 <MenuItem value={0} sx={{ fontStyle: 'italic', height: '2em' }}>
                                     {''}
                                 </MenuItem>
-                                <MenuItem value={EngagementStatus.Published}>Published</MenuItem>
-                                <MenuItem value={EngagementStatus.Closed}>Closed</MenuItem>
+                                <MenuItem value={EngagementDisplayStatus.Open}>Open</MenuItem>
```

Comment:
can we have constants for Open , Upcoming etc

Reply:
I can change the text to constants for you